UI/InGameMenu.h

```diff
@@ -34,8 +35,9 @@ class InGameMenu : public CUIWnd {
     std::shared_ptr<GG::Button> m_save_btn;         //!< Save game button;
     std::shared_ptr<GG::Button> m_load_btn;         //!< Load game button;
     std::shared_ptr<GG::Button> m_options_btn;      //!< Options button;
+    std::shared_ptr<GG::Button> m_concede_btn;      //!< Concede button;
     std::shared_ptr<GG::Button> m_done_btn;         //!< Done button;
-    std::shared_ptr<GG::Button> m_exit_btn;         //!< Quit game button;
+    std::shared_ptr<GG::Button> m_resign_btn;       //!< Quit game button;
```

Comment:
Comment disagrees with the code.  Remove the comment it just repeats the variable name.

Reply:
Removed comments.